civictechprojects/static/css/partials/_ProjectCard.scss

```diff
@@ -98,3 +99,11 @@
   padding: 0.25rem 0.5rem;
   margin: 0.25rem 0.5rem 0.25rem 0;
 }
+
+.ProjectCard-favorite {
+  position: absolute;
+  top: 0px;
+  right: 0px;
+  padding-right: 5px;
+  padding-top: 5px;
```

Comment:
Instead of absolute/top 0/right 0, try
```css
float: right;
padding-top: 4px;
padding-right: 4px;
```
That should position this correctly and let text wrap underneath, though in most cases text will not wrap under the heart since it needs three lines to do so. Definitely worth testing on narrow mobile devices.